### PR TITLE
Misc. patches and improvements

### PR DESCRIPTION
**Pull request by ThatNerdyPikachu**
- Fixes overlay removal yet again
- Adds additional overlay option (disables Pixel accent but keeps everything else)
- Downgrades Unity to v2.2, the latest official release. (v2.3 is from a testing branch, and is not officially released at this time.)

### CODE REVIEW

**Comment by thehappydinoa** [comment]
Wait why do you repeat the overlay prompt?

**Comment by ThatNerdyPikachu** [comment]
> Wait why do you repeat the overlay prompt?

I added an additional option

File: common/install.sh [hunk 1]
```diff
@@ -90,16 +90,28 @@ $FUNCTION "DOWN"
...
 if $FUNCTION; then
   ui_print " "
   ui_print "   Enabling overlays..."
 else
   ui_print " "
-  ui_print "   Disabling overlays..."
-  sed -i -e 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
-  rm -rf /data/resource-cache
-  ui_print "You may want to clear Dalvik cache too!"
+  ui_print " - Overlay Options -"
```
Comment (thehappydinoa) [inline common/install.sh:101]: This doesn't need to prompt the user twice

File: common/install.sh [hunk 2]
```diff
@@ -90,16 +90,28 @@ $FUNCTION "DOWN"
...
+  
+  if $FUNCTION; then
+    ui_print " "
+    ui_print "   Enabling overlays..."
+    sed -i -e 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
+  else
+    ui_print " "
+    ui_print "   Disabling overlays..."
+    sed -i -e 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
+    rm -f $INSTALLER/system/vendor/overlay/Pix3lify.apk
```
Comment (thehappydinoa) [inline common/install.sh:113]: Shouldn't totally kill

File: common/install.sh [hunk 3]
```diff
@@ -90,16 +90,28 @@ $FUNCTION "DOWN"
...
-  rm -rf /data/resource-cache
-  ui_print "You may want to clear Dalvik cache too!"
+  ui_print " - Overlay Options -"
+  ui_print "   Do you want overlays (rounded corners) enabled?"
+  ui_print "   Vol Up = Yes, Vol Down = No"
+  
+  if $FUNCTION; then
+    ui_print " "
+    ui_print "   Enabling overlays..."
+    sed -i -e 's/ro.boot.vendor.overlay.theme/# ro.boot.vendor.overlay.theme/g' $INSTALLER/common/system.prop
```
Comment (thehappydinoa) [inline common/install.sh:108]: You disable on enable?

**Review by thehappydinoa — CHANGES_REQUESTED** [review]

File: common/install.sh (hunk 2 above)
Comment (lzlrd) [reply common/install.sh:113]: @thehappydinoa You sure? It's "disabling overlays" not disabling the theme so I would say that this is correct in my opinion. I think I got an idea tho. Keep tabs on the PRs.

File: common/install.sh (hunk 1 above)
Comment (ThatNerdyPikachu) [inline common/install.sh:101]: @Stallix Do you agree with this?

File: common/install.sh (hunk 2 above)
Comment (ThatNerdyPikachu) [reply common/install.sh:113]: wdym

File: common/install.sh [hunk 4]
```diff
@@ -90,16 +90,28 @@ $FUNCTION "DOWN"
 # fi
 
 ui_print " "
-ui_print " - Select Option -"
-ui_print "   Do you want overlays (theme accent and rounded corners) enabled:"
-ui_print "   Vol Up = Yes, Vol Down = No"
+ui_print " - Overlay Options -"
+ui_print "   Do you want overlays (theme accent and rounded corners) enabled?"
+ui_print "   Vol Up = Yes, Vol Down = More Options"
```
Comment (thehappydinoa) [inline common/install.sh:95]: This is quite helpful

File: common/install.sh (hunk 1 above)
Comment (joeyhuab) [reply common/install.sh:101]: Yes is cool

**Review by lzlrd — COMMENTED** [review]
I'm gonna try do a fixed up version of what you did.

**Comment by lzlrd** [comment]
Re-done at https://github.com/Magisk-Modules-Repo/Pix3lify/pull/41.

**Pull request closed without merging**